openwisp_controller/config/who_is/handlers.py

```diff
@@ -0,0 +1,31 @@
+from django.db.models.signals import post_delete, post_save
+from swapper import load_model
+
+from .. import settings as app_settings
+
+
+def connect_who_is_handlers():
+    # if WHO_IS_CONFIGURED is False, we do not connect the handlers
+    # and return early
+    if not app_settings.WHO_IS_CONFIGURED:
+        return
```

Comment:
Will we reach this stage if WHO_IS_CONFIGURED is false?

Reply:
Yes, this function is executed on config app ready.